source/retryable-writes/tests/insertOne-errorLabels.yml

```diff
@@ -0,0 +1,80 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
```

Comment:
Yes it'll definitely be a different release, but I don't think server team knows which yet. I'll change it to 4.3.x for easy grep-replace.

And yes this is just one example. I wanted your feedback before adding a bunch of new tests so that I didn't go too far off in the wrong direction.